test/appium/tests/atomic/chats/test_group_chat.py

```diff
@@ -170,3 +176,32 @@ def test_add_new_group_chat_member(self):
             self.errors.append('Message that was sent after device 2 has joined is visible')
 
         self.verify_no_errors()
+
+    @marks.testrail_id(5756)
+    @marks.high
+    def test_decline_invitation_to_group_chat(self):
+            self.create_drivers(2)
+            leave_message = 'left the group'
+            message_for_device_2 = 'This message should not be visible for device 2'
+
+            device_1_home, device_2_home = create_users(self.drivers[0], self.drivers[1])
+            chat_name = device_1_home.get_public_chat_name()
+            device_1_chat, device_2_chat = create_new_group_chat(device_1_home, device_2_home, chat_name)
+            device_2_chat.decline_invitation_button.click()
+
+            # device 2: check that chat is deleted and user is redirected to Home view
+            home_view = HomeView(self.drivers[1])
```

Comment:
Unnecessary assignment. device_1_home already has HomeView (there is a create_user() which has get_home_view that has `return HomeView`)